README.md

```diff
@@ -9,3 +9,23 @@ This lets you have multiple Angular microfrontends coexist within the same page.
 ## Documentation
 
 See [documentation on single-spa.js.org](https://single-spa.js.org/docs/ecosystem-angular.html).
+
+## Usage
```

Comment:
The documentation for this project is at https://single-spa.js.org/docs/ecosystem-angular, and can be modified at https://github.com/single-spa/single-spa.js.org/edit/master/website/versioned_docs/version-5.x/ecosystem-angular.md. Can you update that file instead of changing this file here?